server/routes/newSeats.js

```diff
@@ -0,0 +1,76 @@
+// Get dependencies
+const express = require('express');
+const router = express.Router();
+const mongodb = require('mongodb');
+const post = require()
+
+router.get('/', (req, res) => {
+res.send([]);
+    // grab members from cses page
+    var MongoClient = mongodb.MongoClient;
+
+    var newSeats = new mongoose.Schema({
+      //var  type
+      room: Number,
+      seatNumber: Number
+  });
+
+    //mongoose schema into a mongoose model
+    var Seats = mongoose.model("Seats", newSeats)
+
+    MongoClient.connect(DB_URL,function(err,db){
```

Comment:
Our connect code should actually be stored in a provider and required from within server.js 

See app.js and EmployeeProvider.js in this repo:
https://github.com/cloudControl/nodejs-express-mongodb-example-app